bootcd/rpms/genesis_scripts/src/genesis.init

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+#
+# chkconfig: 2345 09 89
+# description: run genesis bootloader
+#
+
+### BEGIN INIT INFO
+# Required-Start:    $local_fs $network $syslog sshd udev-post
+# Default-Start:     2 3 4 5
+# Description:       run genesis bootloader
+### END INIT INFO
+
+. /etc/rc.d/init.d/functions
+
+case "$1" in
+    start)
+	echo -n $"Starting genesis bootloader:  "
+	/usr/bin/run-genesis-bootloader &>/dev/null &
```

Comment:
This doesn't accurately daemonize the process I think. We aren't closing stdin, and this process is still in the parent's process group (nohup/daemonize will double fork to escape the current pgroup). I think this will be ok when executed by init because the pgroup won't be different, but if a user ran the start script from a terminal, exiting the terminal will send a term signal to this backgrounded process (I think). Have you tested this behavior?